meilisearch/models.py

```diff
@@ -0,0 +1,46 @@
+from dataclasses import dataclass
+from typing import Any, List, Dict, Optional
+from camel_converter.pydantic_base import CamelBase
+
+@dataclass
+class DocumentsResults:
+    results: List[Dict[str, Any]]
+    offset: int
+    limit: int
+    total: int
+
+class Task(CamelBase):
+    uid: Optional[str]
+    index_uid: str
+    task_uid: Optional[str]
```

Comment:
The class `Task` should be used for the return of methods who return an object `Task` from the `Meilisearch` API only this 3 one: `get_task()`, `get_tasks()` and `wait_for_task()`
```suggestion
class Task(CamelBase):
    uid: str
    index_uid: str
```

Reply:
Thanks for clarifying 💪🏻 Confused me as wait_for_task and the others mention returning an asynchronous task: https://github.com/meilisearch/meilisearch-python/blob/bca102c8e9f87220725582f7df2d224684eb0030/meilisearch/index.py#L190-L191